### ISSUE TITLE
Scrollbar warning feels out of place

### ISSUE THREAD

**Issue by usernamehw** [maoma87/NightWolfTheme#5]
Default green color for warnings doesn't look very good

```json
"editorOverviewRuler.warningForeground": "#128812b3",
```

**Comment by usernamehw** [comment]
Warning in explorer has that default green too
```json
"list.warningForeground"
```

**Comment by maoma87** [comment]
fixed in v1.1.0

**Comment by maoma87** [comment]
thanks!
